### ISSUE TITLE
Loss calculation of the unlabeled data

### ISSUE THREAD

**Issue by yunzhewang** [iBelieveCJM/pseudo_label-pytorch#6]
`# make the pseudo label for unlabeled data
output2 = net(img2);
_, label2 = torch.max(output2, 1);
            
#semi-supervised loss
loss = criterian(output1, label1) + alpha * criterian(output2, label2);`

Could you please explain why criterian() for unlabeled data works? Because  label2 is actually calculated from output2, they are the same indeed? Why non-zero loss is obtained?
